station-worldgen-api-v0/src/main/java/net/modificationstation/stationapi/api/worldgen/biome/BiomeProviderRegisterEvent.java

```diff
@@ -0,0 +1,10 @@
+package net.modificationstation.stationapi.api.worldgen.biome;
+
+import net.mine_diver.unsafeevents.Event;
+import net.mine_diver.unsafeevents.event.EventPhases;
+import net.modificationstation.stationapi.api.StationAPI;
+
+@EventPhases(StationAPI.INTERNAL_PHASE)
+public class BiomeProviderRegisterEvent extends Event {
+	public BiomeProviderRegisterEvent() {}
```

Comment:
Just noticed this, you should probably use `@SuperBuilder` instead of a normal constructor, unless you really need the constructor for some reason.

Makes it more future-proof if you might want to add fields to the event class in the future. Unless you actually never want to add anything to it.

Also, more compliant with the rest of StAPI events.

Reply:
I added that annotation, should be fine now